frontend/src/views/MLStudio/MLStudioEditForm.js

```diff
@@ -89,7 +89,7 @@ const PipelineEditForm = (props) => {
           variant: 'success'
         });
         navigate(
-          `/console/mlstudio/${response.data.updateUserProfile.sagemakerStudioUserProfileUri}`
```

Comment:
Is this `updateUserProfile` still used? Could not find anything related to it in the backend